bitnami/cassandra/Chart.yaml

```
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: cassandra
-version: 6.0.3
+version: 6.0.4
```

```suggestion
version: 6.0.5
```
as a new version appeared in the meantime (https://github.com/bitnami/charts/pull/4132)